manifests/config.pp

```diff
@@ -168,8 +168,14 @@
         ssl_content => template('foreman/lookup_identity.conf.erb'),
       }
 
-      foreman::config::apache::fragment { 'auth_kerb':
-        ssl_content => template('foreman/auth_kerb.conf.erb'),
+      if $facts['os']['family'] == 'RedHat' && versioncmp($facts['os']['release']['major'], '8') >= 0 {
```

Comment:
technically, we could also use the same on debian, as it has mod_gssapi too.

and actually, EL7 has it too, so probably let's switch to it completely?

Reply:
I did that now, fuck that mod_auth_kerb